src/App.js

```diff
@@ -1,30 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { Switch, Route } from 'react-router-dom';
 import FirebaseTesting from './pages/FirebaseTest';
 import FooterTabs from './components/FooterTabs';
 import ShoppingList from './pages/ShoppingList';
 import AddItem from './pages/AddItem';
+import NewListButton from './components/newListButton';
+import * as ls from 'local-storage';
 
 function App() {
+	const [token, setToken] = useState(ls.get('shoppingListToken'));
+	console.log('token', ls.get('shoppingListToken'), 'state', token);
+
```

Comment:
I like this solution! I just learned about a cool way to use destructuring to log things out. So, previously, I might do something like this: `console.log('token', token)` .  A slick way to do it is `console.log({token})` . This will print out `{ token: whateverTokenIs }` to the console.